dashboard.org

```diff
@@ -53,9 +53,9 @@ Call [[elisp:mu4e-dashboard-deactivate][mu4e-dashboard-deactivate]] before editi
 #+STARTUP: showall showstars indent
 
 #+NAME: query
-#+BEGIN_SRC shell :results list raw :var query="flag:unread count=5 
+#+BEGIN_SRC sh :results list raw :var query="flag:unread" count=5 
```

Comment:
I thought we we supposed to use `shell` now instead of `sh`.

Reply:
"shell" is the user configured $SHELL whereas "sh" is /bin/sh which should always be a POSIX compliant shell. You can actually use any installed shell if it's setup in the org languages section.

Reply:
Ok. I think I got confused by `ob-sh` being renamed `ob-shell` and thus I assumed `shell` was the proper way.